src/components/Dashboard.js

```diff
@@ -4,487 +4,484 @@ class Dashboard extends Component {
-                                                          </div>
-                                                      </td>
-                                                      <td>Contracting</td>
-                                                      <td className="text-right">
-                                                          690
-                                                      </td>
-                                                      <td className="text-right">
-                                                          7.87%
-                                                      </td>
-                                                  </tr>
-                                                  <tr>
-                                                      <td>
-                                                          <div className="flag">
-                                                          </div>
-                                                      </td>
-                                                      <td>Data Structures & Algorithms</td>
-                                                      <td className="text-right">
-                                                          600
-                                                      </td>
-                                                      <td className="text-right">
-                                                          5.94%
-                                                      </td>
-                                                  </tr>
-                                                  <tr>
-                                                      <td>
-                                                          <div className="flag">
-                                                          </div>
-                                                      </td>
-                                                      <td>Technical Interviewing</td>
-                                                      <td className="text-right">
-                                                          550
-                                                      </td>
-                                                      <td className="text-right">
-                                                          4.34%
-                                                      </td>
-                                                  </tr>
-                                              </tbody>
+                                            <tbody>
+                                              <tr>
+                                                <td>
+                                                  <div className="flag">
+                                                  </div>
+                                                </td>
+                                                <td>Advanced Web Patterns</td>
```

Comment:
Are these all hard coded for just a demo for our current dashboard?

Reply:
The whole Dashboard component is just a fake mockup.  Nothing connected to backend yet.

Reply:
I'd recommend to put all of this mock up data into a JSON object at the top of the file and then pull it into the render through the object. 

```
const myObject= {
  titleOfRubric: "Some Title",
  ...
}

in the render:
{ myObject.titleOfRubric }
```

That way, when you get the real data, you A) know the exact structure of the data you need to get & B) you will just have to swap out the object and you're done!

Reply:
We still have to break up this view into more granular components than this.  Each subcomponent will get instantiated with all of that info passed in to its props.